Makefile.am

```diff
@@ -235,7 +235,7 @@ bots:
 	tools/make-bots
 
 multi-word: po/cockpit.pot
-	$(MSGGREP) --no-wrap --msgid -e '\S\+\s' po/cockpit.pot | grep "^msgid " | cut -d " " -f2- | sort --reverse > titles
+	msggrep --no-wrap --msgid -e '\S\+\s' po/cockpit.pot | grep "^msgid " | cut -d " " -f2- | sort --reverse > titles
```

Comment:
FTR, I'm not actually sure why the previous configure.ac did that "static path" detection. There was no way to override it with some `--with` option or so. :+1: on that simplification.